### ISSUE TITLE
Error: too many values to unpack (expected 2)

### ISSUE THREAD

**Issue by MushMello** [pixray/pixray#72]
When I want to run a prediction, I get an error for line 493 in pixray.py:

    batch1, transforms1 = self.augs_zoom(torch.cat(cutouts[:self.cutn_zoom], dim=0))

That there are too many values to unpack. Changing the dimension to 1 doesn't give me the error, but rather gives me an error a little bit later in __init__.py from Kornia that the tensor is not rgb or grayscale

Edit: Now it suddenly tells me "not enough packages" when the dimension is 1, but 0 is already too many again

**Comment by dribnet** [comment]
can you check you are using `kornia==0.6.2`
